### ISSUE TITLE
Plotting recipe: markers are broken

### ISSUE THREAD

**Issue by mzgubic** [invenia/Intervals.jl#134]
```
using Intervals
using Plots
intervals = [Interval{Closed, Closed}(float(x), float(x + 0.5)) for x in 1:11]
plot(intervals, 1:11)
```

gives the following
<img width="594" alt="image" src="https://user-images.githubusercontent.com/15211548/88075225-c9c70980-cb78-11ea-8dc0-04a55a469381.png">

I suspect what happened is that the markers used to be filtered by NaN positions, and are not anymore. Will give it a go.

**Comment by mzgubic** [comment]
The problem comes from using Plots 1.5.6 rather than 0.29.9.
https://github.com/JuliaPlots/Plots.jl/issues/2876

**Comment by AlexRobson** [comment]
I know this thread is old, but do you recall what the expected behaviour should be here? Is the issue that the markers are now showing up for the interval at 3, 6 and 9? I'm just revisiting https://github.com/invenia/Intervals.jl/pull/165 which is related.

**Comment by mzgubic** [comment]
Yes, they are *not* showing up at 3, 6, 9. It would also be nice if edges were different for open/closed, but not sure that's that's in scope :)
